components/script/dom/request.rs

```diff
@@ -295,10 +296,9 @@ impl Request {
                 Some(s) => s,
                 None => return Err(Error::Type("Method is not a valid UTF8".to_string())),
             };
-            let normalized_method = normalize_method(method_string);
+            let method = normalize_method(method_string);
             // Step 25.3
-            let hyper_method = normalized_method_to_typed_method(&normalized_method);
-            *request.method.borrow_mut() = hyper_method;
+            *request.method.borrow_mut() = method;
```

Comment:
Is `method` used anywhere else in this scope? If not, we can just make the above two lines into one.